draft-pauly-httpbis-geoip-hint.md

```diff
@@ -73,63 +96,99 @@ For example, the header for an entry "192.0.2.5,US,US-AL,Alabaster" would be:
     Sec-CH-IP-Geo = "US,US-AL,Alabaster"
 ~~~
 
-This field also defines a parameter, "feed", that contains the URI of the
-IP geolocation feed that is authoritative for this entry. For example:
-
-~~~
-    Sec-CH-IP-Geo = "SG,SG-01,Singapore"; feed="https://noc.ietf.org/geo/google.csv"
-~~~
-
-Servers that can provide different content based on Geohash hints SHOULD include
-the headers in their "Accept-CH" list.
+Given that the Sec-CH-Geo is a high-entropy client hint, meaning it is
```

Comment:
The examples here include `IP-` in the header name, but the text does not. Since the draft emphasizes that the client should determine the location from its own IP, maybe it makes sense to keep the header name `Sec-CH-IP-Geo`. I don't have strong opinions, and it's probably too early for long discussion on the topic, but the draft should at least be consistent.